Sources/BranchSDK/Public/Branch.h

```diff
@@ -568,7 +568,7 @@ extern NSString * __nonnull const BNCSpotlightFeature;
  */
 + (void)enableLogging;
 + (void)enableLoggingAtLevel:(BranchLogLevel)logLevel withCallback:(nullable BranchLogCallback)callback;
-+ (void)enableLoggingAtLevel:(BranchLogLevel)logLevel withAdvancedCallback:(nullable BranchAdvancedLogCallback)callback;
++ (void)enableLoggingAtLevel:(BranchLogLevel)logLevel withAdvancedCallback:(BranchAdvancedLogCallback)callback;
```

Comment:
Since this is no longer nullable, do we expect any existing integrations to break as a result? What is the impact?

Reply:
@gdeluna-branch  ` + (void)enableLoggingAtLevel:(BranchLogLevel)logLevel withCallback:(nullable BranchLogCallback)callback; ` has `nullable` attribute. So this will take care of any code calling `enableLoggingAtLevel` without callback.
Both these API just differ from the callback param - one is older one and the other one is `BranchAdvancedLogCallback` which was introduced in last Release. So just changed the attiribute type of this new API. Nothing is changed for the older one.